### ISSUE TITLE
Where are these three variables defined?  Should they be strings?

### ISSUE THREAD

**Issue by cclauss** [ColdGrub1384/Pyto#19]
[flake8](http://flake8.pycqa.org) testing of https://github.com/ColdGrub1384/Pyto on Python 3.7.1

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./Pyto_Mac/Startup.py:9:25: F821 undefined name 'app'
__builtins__.__host__ = app
                        ^
./PytoCore/Startup.py:3:29: F821 undefined name 'iOS'
__builtins__.__platform__ = iOS
                            ^
./Pyto/Startup.py:7:25: F821 undefined name 'app'
__builtins__.__host__ = app
                        ^
3     F821 undefined name 'app'
3
```
__E901,E999,F821,F822,F823__ are the "_showstopper_" [flake8](http://flake8.pycqa.org) issues that can halt the runtime with a SyntaxError, NameError, etc. These 5 are different from most other flake8 issues which are merely "style violations" -- useful for readability but they do not effect runtime safety.
* F821: undefined name `name`
* F822: undefined name `name` in `__all__`
* F823: local variable name referenced before assignment
* E901: SyntaxError or IndentationError
* E999: SyntaxError -- failed to compile a file into an Abstract Syntax Tree

**Comment by ColdGrub1384** [comment]
These variables are defined just before. It's for platform checking. They are just strings, but I defined them so scripts can check for platform like that: `__platform__ is iOS` or `__platform__ is macOS`. They are defined on `__builtins__`, so this commit should fix it: 8cd443c9aa0a7d69c27876301b61a2aa470ed826

**Comment by cclauss** [comment]
Nice.
